README.md

```diff
@@ -83,28 +83,28 @@ slotinfo get-state-id --state_name Maharashtra
 slotinfo get-district-id --state_id 21 --district_name Pune
 
 3. Check available appointment slots district wise
-slotinfo district-wise --district_id 334 --date 10-05-2021 --age_filter 18 --notify_on whatsapp
+slotinfo district-wise --district_id 334 --date 10-05-2021 --age_filter 18 --notify_on whatsapp --vaccine_type covishield --vaccine_type covaxin
```

Comment:
why do we have to specify vaccine_type multiple times  `--vaccine_type covishield  --vaccine_type covaxin` ?

Reply:
this is because we dont know no of args at first. there is option to add nargs=count but after but after some days there can be some other vaccines too. so it is anticipated here

Reply:
yeh, just tested with nargs it requires to specify all the args so that won't work. Let's keep this for now.